scripts/build_server.sh

```diff
@@ -0,0 +1,10 @@
+echo "Build server files..."
+echo "Building CoffeeScript..."
+./node_modules/.bin/coffee -cb --output build/server server
```

Comment:
Indeed, good idea. Altough, I have some concerns about it:
* Using `export` seems to be overkill, as it will pass down the `$PATH` to subprocesses, while we only need it in the current script I think?
* I think we should append to `$PATH` rather than prepend, in order to avoid having a system executable being replaced silently. This however means that the system-wide binaries will be preferred.